R/scrape.population.distribution.R

```diff
@@ -36,22 +32,23 @@ scrape.population.distribution = function(year, league = c('mens', 'womens')) {
 
   results = data.frame(names = names, probabilities = probabilities, round = as.numeric(round)) %>%
       tidyr::spread(round, probabilities) %>%
-      dplyr::mutate(names = as.character(names)) %>%
-      dplyr::mutate(names = ifelse(names == 'BON/LA', 'BON/UCLA',
-                            ifelse(names == 'NCC/TS', 'NCC/Texas Southern',
-                            ifelse(names == 'ASU/SJU', "ASU/St John's",
-                            ifelse(names == 'MSM/TXSO', 'MSM/Texas Southern',
-                            ifelse(names == 'MSU/UCLA', 'Michigan State/UCLA',
-                            ifelse(names == 'WICH/DRKE', 'Wichita State/Drake',
-                            ifelse(names == 'Florida State', 'FSU',
-                            ifelse(names == 'Kansas State', 'KSU',
-                            ifelse(names == 'North Carolina', 'UNC',
-                            ifelse(names == 'Ohio State', 'OSU',
-                            ifelse(names == 'UNC Greensboro', 'UNCG',
-                            ifelse(names == 'Virginia', 'UVA',
-                            ifelse(names == 'New Mexico State', 'New Mexico St',
-                            ifelse(names == 'South Dakota State', 'South Dakota St', names))))))))))))))) %>%
-      dplyr::mutate(names = as.factor(names))
+      dplyr::mutate(names = as.character(names)) #%>%
+      # NOTE (SP): Why did we do this??
+#      dplyr::mutate(names = ifelse(names == 'BON/LA', 'BON/UCLA',
+#                            ifelse(names == 'NCC/TS', 'NCC/Texas Southern',
+#                            ifelse(names == 'ASU/SJU', "ASU/St John's",
+#                            ifelse(names == 'MSM/TXSO', 'MSM/Texas Southern',
+#                            ifelse(names == 'MSU/UCLA', 'Michigan State/UCLA',
+#                            ifelse(names == 'WICH/DRKE', 'Wichita State/Drake',
+#                            ifelse(names == 'Florida State', 'FSU',
+#                            ifelse(names == 'Kansas State', 'KSU',
+#                            ifelse(names == 'North Carolina', 'UNC',
+#                            ifelse(names == 'Ohio State', 'OSU',
+#                            ifelse(names == 'UNC Greensboro', 'UNCG',
+#                            ifelse(names == 'Virginia', 'UVA',
+#                            ifelse(names == 'New Mexico State', 'New Mexico St',
+#                            ifelse(names == 'South Dakota State', 'South Dakota St', names))))))))))))))) %>%
```

Comment:
My memory is we did this for backwards compatibility for names changing in the population distribution data over time. Most often it was related to play-in teams, because ESPN's population pick table would squeeze characters in years where a team was a play-in team but spell out the full team name in years where they were a standalone team. In the other cases it was just inconsistency over time in their data, where our mapping table approach would only support one population team name for all years.

Reply:
I did notice that some of the population pick names were different from what we'd saved down. Do you think we should keep this line in there? Right now, the code just overwrites the mapping with the new one, unfortunately.

Reply:
I'll leave it as is (ie commented out). We may want to implement a better way to achieve backward compatibility in the future, but I think that's not an important feature, and I don't like this way of doing it.

Reply:
Makes sense, sounds good